src/user/SignUpForm.js

```diff
@@ -0,0 +1,80 @@
+// Sign Up Form allows anyone to sign up for an account to access their personal dashboards
+// The contact form requests First Name, Last Name, Email, Password & Confirm Password.
+// The sign up form then redirects the user to their personal dashboards
+
+import React from 'react';
+import {Form, Button} from 'react-bootstrap';
+
+class SignUpForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+
+    this.submitHandler = this.submitHandler.bind(this);
+    this.handleFormChange = this.handleFormChange.bind(this);
+    this.passwordValidator = this.passwordValidator.bind(this);
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.confirmPassword !== this.state.confirmPassword) {
+      this.passwordValidator();
+    }
+  }
+
+  passwordValidator() {
+    this.setState({
+        errorMessage: this.state.password === this.state.confirmPassword ? '' : 'Your password does not match'
+      })
+  }
+  
+  submitHandler(e) {
+    e.preventDefault();
+    // insert submission logic here
+  }
+
+  handleFormChange(e) {
+    const {currentTarget} = e;
+    this.setState({
+      [currentTarget.name]: currentTarget.value,
+    });
+  }
+
+  render() {
+    return (
+      <div className="form-component-container">
+        <h1>Sign Up!</h1>
+        <Form className="contact-form" onSubmit={this.submitHandler}>
```

Comment:
onSubmit not needed as it is being handled by the onClick function in the submit button
```suggestion
        <Form className="contact-form">
```

Reply:
Right on, don't need this redundant code anymore. Removed.